content/posts/November-2021.md

```diff
@@ -30,3 +30,13 @@ weight: 69
   The action is supposed to generate whole changelog entry (including `- ` at
   the start of the lines) and has a priority over any other way we modify the
   changelog with. ([packit#1367](https://github.com/packit/packit/pull/1367))
+
+## Week 46 (November 15th - November 19th)
+
+- A new config option `env` has been added for specifying environment variables.
+  ([packit#1411](https://github.com/packit/packit/pull/1411))
+- Environment variables from config are now passed to Testing Farm
+  along with some more pre-defined variables (e.g. name of the project, URL, etc).
+  ([packit-service#1275](https://github.com/packit/packit-service/pull/1275))
```

Comment:
What about merging those two to make sure that this is one feature and used only for TF?